### PR TITLE
Issue #11758: Adding CheckForVariable function to util.sh

### LINKED ISSUE #11758: Adding CheckForVariable function to util.sh
Whenever we add a token or key, it is important to check if it exists before we run the shell script.

Currently, I am using this function in the release project but as we use a lot of tokens in different shell scripts, we need to make a general function in util.sh so that we can call this function and remove repetition of creating this function.

Function:

```
checkForVariable() {
    if [[ "$(declare -p "$1")" =~ ^declare\ -x ]]; then
       echo "Error: Define $1 environment variable"
       exit 1
    fi
}
```

Usage:
`checkForVariable "TOKEN_NAME"`

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Fixes #11758

### CODE REVIEW

File: .ci/prepare-settings.sh [hunk 1]
```diff
@@ -4,14 +4,6 @@ set -e
 mkdir -p .ci-temp
 cp .ci/release-settings.xml .ci-temp/
 
-checkForVariable() {
```
Comment (romani) [inline .ci/prepare-settings.sh:7]: Add `source ./.ci/util.sh`

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .ci/prepare-settings.sh (hunk 1 above)
Comment (Rahulkhinchi03) [reply .ci/prepare-settings.sh:7]: Done changes.

Comment (romani) [reply .ci/prepare-settings.sh:7]: Not done, recheck push and share testing of this script

Comment (Rahulkhinchi03) [reply .ci/prepare-settings.sh:7]: Done

Comment (Rahulkhinchi03) [reply .ci/prepare-settings.sh:7]: Done

Comment (Rahulkhinchi03) [reply .ci/prepare-settings.sh:7]: sdfjnsd

Testing: Getting the ERROR of checkForVariable. Working.

Comment (romani) [reply .ci/prepare-settings.sh:7]: Now I see `source`

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**